src/nidcpower/metadata/enums.py

```diff
@@ -566,6 +566,87 @@
             },
         ],
     },
+    'Signals': {
```

Comment:
Where did this term Signals come from? Is it pre-established for NI-DCPower or another one of the APIs for some other ADE? Or was it derived from "niDCPower_ExportSignal"?

I do see several APIs have it.

Reply:
Derived from niDCPower_ExportSignal. should it be changed to Export Signal ?

Reply:
It should be derived directly from the corresponding typedef name in LabVIEW. If you find yourself having to come up with names / documentation, then we need to be careful.

Reply:
All,

We need to have the names and descriptions of the enums come from the LabVIEW API rather than the C API. This is because all of our extraction for enums already comes from the LabVIEW API.

This will cause a disconnect in the documentation, which we need to look at in #502.

Please update the enums to follow the LabVIEW naming and documentation conventions.

Reply:
Updated internal files to have names following LabVIEW naming convensions, instead C.

Should we change it here since it is generated. Will extraction tool take care of this?

Reply:
Extraction tool will generate the new files, but somebody needs to commit them to GitHub. @texasaggie97 usually handles it.